manifest/manifest.go

```diff
@@ -24,10 +24,17 @@ const (
 	DockerV2Schema2ConfigMediaType = "application/vnd.docker.container.image.v1+json"
 	// DockerV2Schema2LayerMediaType is the MIME type used for schema 2 layers.
 	DockerV2Schema2LayerMediaType = "application/vnd.docker.image.rootfs.diff.tar.gzip"
+	// DockerV2Schema2LayerMediaTypeZstd is the MIME type used for schema 2 layers compressed with zstd.
+	DockerV2Schema2LayerMediaTypeZstd = "application/vnd.docker.image.rootfs.diff.tar.zstd"
+	// DockerV2SchemaLayerMediaTypeUncompressed is the mediaType used for uncompressed layers.
+	DockerV2SchemaLayerMediaTypeUncompressed = "application/vnd.docker.image.rootfs.diff.tar"
 	// DockerV2ListMediaType MIME type represents Docker manifest schema 2 list
 	DockerV2ListMediaType = "application/vnd.docker.distribution.manifest.list.v2+json"
 	// DockerV2Schema2ForeignLayerMediaType is the MIME type used for schema 2 foreign layers.
 	DockerV2Schema2ForeignLayerMediaType = "application/vnd.docker.image.rootfs.foreign.diff.tar.gzip"
+	// OCIV1MediaTypeImageLayerZstd is the MIME types used for OCI v1 layers compressed with zstd.
+	// Note that this MIME type is not (yet) part of the OCI v1 image specification.
+	OCIV1MediaTypeImageLayerZstd = "application/vnd.oci.image.layer.v1.tar+zstd"
```

Comment:
Hum, https://github.com/opencontainers/image-spec/pull/788 has added the textual specification but not the Go constant.

Reply:
Opened a PR to add them.